File: templates/matlab/microgrid_market_state_map.m
function fig = microgrid_market_state_map()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('heatmap', 3803, 'microgrid and market analysis: state heatmap', 'microgrid and market analysis', 'state heatmap');
end

File: templates/python/microgrid_market_state_map.py
"""microgrid_market_state_map: 微电网与市场状态热力图（heatmap 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='microgrid and market analysis: state heatmap'):
    return make_template_figure('heatmap', seed=3803, title=title, domain='microgrid and market analysis', topic='state heatmap')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
